README.md

```diff
@@ -30,6 +30,252 @@
 
 # Triton Inference Server
 
-**NOTE: You are currently on the r20.10 branch which tracks
-stabilization towards the next release. This branch is not usable
-during stabilization.**
+Triton Inference Server provides a cloud and edge inferencing solution
+optimized for both CPUs and GPUs. Triton supports an HTTP/REST and
+GRPC protocol that allows remote clients to request inferencing for
+any model being managed by the server. For edge deployments, Triton is
+available as a shared library with a C API that allows the full
+functionality of Triton to be included directly in an
+application.
+
+## What's New in 2.4.0
+
+* A new Python backend allows Python code to run as a model within Triton. See 
+  https://github.com/triton-inference-server/python_backend.
+
+* A new Python backend allows DALI pipelines to run within Triton. See 
```

Comment:
This should say "A new backend allows..."

Reply:
Then it needs to be changed in the Release Notes as well.

Reply:
Should it be "A new DALI backend.."